xtevent/get_unit_time_effects.ado

```diff
@@ -1,4 +1,4 @@
-*! get_unit_time_effects.ado 2.2.0 March 15 2023
+*! get_unit_time_effects.ado 3.0.0 January, 2024

```

Comment:
@Constantino-Carreto-Romero day pending

Reply:
updated release date in ado files and help files in https://github.com/JMSLab/xtevent/pull/168/commits/a7c402563f6235798efe08e3d477aff843e83446